CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.3.0 (unreleased)
```

Comment:
This file will be handled prior to release to prevent merge conflicts across PRs: https://github.com/hashicorp/terraform-plugin-framework-validators/blob/main/.github/CONTRIBUTING.md#releases

Reply:
Removed changes from `CHANGELOG.md`.